### ISSUE TITLE
worker.js setClientId error

### ISSUE THREAD

**Issue by dcollinsf5** [iarna/abraxas#14]
Hello,
I've encountered an issue using Abraxas that I believe may be a simple typo in workers.js:
https://github.com/iarna/abraxas/blob/master/worker.js#L43
This line references 'id' but there is no 'id' variable defined in that scope.

I set client id in the connect callback, using a promisified (by bluebird) connect method, like so:
```javascript
Abraxas.Client.connectAsync(jobServerConf)
  .then(function(client) { client.setClientId('Worker1'); });
```
Strangely, in my development environment where Gearman is on my local machine, this works fine 100% of the time, but in production with added network latency, the same code throws the above mentioned error 100% of the time.

I noticed there's no eslintrc or jshintrc file for the project; do you use static analysis of any kind for Abraxas?

**Comment by dcollinsf5** [comment]
We forked a copy of the repo and adjusted L43 to use self._clientId in it's setClientId() call.  This seems to have fixed the problem. FYI, this error was also being thrown any time a worker reconnected to a Gearman server if setClientId() was called in the .on('connect') handler.

**Comment by iarna** [comment]
If you do a pull request, I'm quite happy to accept them. I'm not actively working on this project at the moment, but it is something I'd like to see continue to live.

**Comment by iarna** [comment]
Thanks for finding this– the fix is in [abraxas@2.1.1](https://github.com/iarna/abraxas/tree/v2.1.1) on the registry now.
